static_report/src/components/SingleReport/SRTableColumnDetails.tsx

```diff
@@ -1,16 +1,14 @@
-import React from 'react';
 import { Code, Flex, Text } from '@chakra-ui/react';
 import { MetricsInfo } from '../shared/MetrisInfo';
 import { SingleReportSchema } from '../../sdlc/single-report-schema';
 import { formatNumber, getColumnDetails, getMissingValue } from '../../utils';
 
-// Temp Typing
+// FIXME: Temp Typing
 type SRTableColumnDetailsProps = {
-  column: SingleReportSchema['tables']['ACTION']['columns']['DATE'];
+  column: SingleReportSchema['tables']['ACTION']['columns'];
```

Comment:
wouldn't this break the prop type? should be a represented by a `column`, 
hence all the `as string` below.

Reply:
`DATE` is an `unknow` type, removed it currently.